Library/Formula/polarssl.rb

```diff
@@ -29,9 +29,9 @@ def install
     system "make"
     system "make", "install"
     # Why does PolarSSL ship with GNU's Hello included? Let's remove that.
-    rm "#{bin}/hello"
+    rm_f "#{bin}/hello"
     # Remove the pointless example application that hooks into system OpenSSL
-    rm "#{bin}/o_p_test"
+    rm_f "#{bin}/o_p_test"
```

Comment:
Mostly paranoia on my part. Upstream are making some fairly big changes, and if either of these elements gets removed/changed in the ` head ` rm will error out, whereas ` rm_f ` should plow on regardless, I believe.